src/tags.jl

```diff
@@ -0,0 +1,236 @@
+export 
```

Comment:
I would try to avoid this many symbols to the outside world, so maybe we can wrap this as a module:

```julia
module Tags

export ...
const ...
```

So that users can do `using ExifViewer.Tags` to manually load all symbols into the workspace. What do you think?

Reply:
I mean this is a bit of hassle either way:
- So many symbols/tags
- How to best name them?Writing them in tags keyword is quite irritating for me who did just 2 of them
- Make it easier for user to easily access them
  - We can do it either way tbf cause both ways are irritaing enough for the user, automatic load does ease it up slightly but I agree that we are loading a lot of symbols. Does reduction of symbols loaded has any effect on perfomance/compilation time? I can see the design perspective(parts of it atleast)

Reply:
I see... How about defining a normalization method for it:

```julia
normalize_exif_flag(flags::Union{AbstractVector, Tuple}) = map(normalize_exif_flag, flags)
normalize_exif_flag(flag::AbstractString) = getfield(LibExif, Symbol("EXIF_TAG_" * flag))
normalize_exif_flag(flag::LibExif.ExifTag) = flag
normalize_exif_flag(flag::Int) = LibExif.ExifTag(flag)
```